test/Emblem.js

```diff
@@ -8,14 +8,14 @@ var Dat = require('../build/transport/Dat').Dat
 
 describe('Emblem', function(){
     describe('Add Dat', function(){
-        it('should sucessfully add dat to emblem', function(){
+        it('Successfully adds dat to emblem', function(){
```

Comment:
Space before and after parens in this file too

Reply:
done